lib/src/postgres_target.rs

```diff
@@ -266,37 +200,39 @@ impl SlotsProcessing {
         &self,
         client: &postgres_query::Caching<tokio_postgres::Client>,
         update: &SlotUpdate,
+        account_write: AccountWrite,
         meta: &SlotPreprocessing,
     ) -> anyhow::Result<()> {
+        let pubkey = encode_address(account_write.pubkey);
```

Comment:
Nope this is wrong, revert this slot update thread edits. We will have to have separate queries for this slot update thread. This will insert the account writes 2x times because there's a separate tokio thread for inserting the account writes.